.github/labeler.yml

```
@@ -0,0 +1,18 @@
+# .github/labeler.yml
+
+"books":                    # Label for changes related to books
```

```suggestion
books:                    # Label for changes related to books
```

Do we need the quotes ?